### ISSUE TITLE
Failed composer update due to missing services arguments id

### ISSUE THREAD

**Issue by Mathieudewet** [Sylius/Sylius#8178]
```
composer update
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package symfony-cmf/media-bundle is abandoned, you should avoid using it. No replacement was suggested.
Generating autoload files
> Incenteev\ParameterHandler\ScriptHandler::buildParameters
Updating the "app/config/parameters.yml" file
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::buildBootstrap
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache
PHP Fatal error:  Uncaught Symfony\Component\DependencyInjection\Exception\InvalidArgumentException: Tag "<argument>" with type="service" has no or empty "id" attribute in "/var/www/html/sylius-socle/vendor/sylius/sylius/src/Sylius/Bundle/ThemeBundle/DependencyInjection/../Resources/config/services/integrations/assets.xml". in /var/www/html/sylius-socle/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php:484
Stack trace:
#0 /var/www/html/sylius-socle/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php(279): Symfony\Component\DependencyInjection\Loader\XmlFileLoader->getArgumentsAsPhp(Object(DOMElement), 'argument', '/var/www/html/s...', false, false)
#1 /var/www/html/sylius-socle/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php(143): Symfony\Component\DependencyInjection\Loader\XmlFileLoader->parseDefinition(Object(DOMElement), '/var/www/html/s...', Array)
#2 /var/www/html/sylius-socle/vendor/symfony/symfony/src in /var/www/html/sylius-socle/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php on line 484
Script Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache handling the post-update-cmd event terminated with an exception

                                                                                                                                                                  
  [RuntimeException]                                                                                                                                              
  An error occurred when executing the "'cache:clear --no-warmup'" command:                                                                                       
  PHP Fatal error:  Uncaught Symfony\Component\DependencyInjection\Exception\InvalidArgumentException: Tag "<argument>" with type="service" has no or empty "id"  
   attribute in "/var/www/html/sylius-socle/vendor/sylius/sylius/src/Sylius/Bundle/ThemeBundle/DependencyInjection/../Resources/config/services/integrations/ass  
  ets.xml". in /var/www/html/sylius-socle/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php:484                           
  Stack trace:                                                                                                                                                    
  #0 /var/www/html/sylius-socle/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php(279): Symfony\Component\DependencyInje  
  ction\Loader\XmlFileLoader->getArgumentsAsPhp(Object(DOMElement), 'argument', '/var/www/html/s...', false, false)                                               
  #1 /var/www/html/sylius-socle/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php(143): Symfony\Component\DependencyInje  
  ction\Loader\XmlFileLoader->parseDefinition(Object(DOMElement), '/var/www/html/s...', Array)                                                                    
  #2 /var/www/html/sylius-socle/vendor/symfony/symfony/src in /var/www/html/sylius-socle/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader  
  /XmlFileLoader.php on line 484
```

Works after remove them 
-------------------------------

```
<?xml version="1.0" encoding="UTF-8"?>

<!--

 This file is part of the Sylius package.

 (c) Paweł Jędrzejewski

 For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.

-->

<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
    <services>
        <service id="sylius.theme.asset.assets_installer" class="Sylius\Bundle\ThemeBundle\Asset\Installer\AssetsInstaller">
            <argument type="service" id="filesystem" />
            <argument type="service" id="kernel" />
            <argument type="service" id="sylius.repository.theme" />
            <argument type="service" id="sylius.theme.hierarchy_provider" />
            <argument type="service" id="sylius.theme.asset.path_resolver" />
        </service>

        <service id="sylius.theme.asset.assets_installer.output_aware" class="Sylius\Bundle\ThemeBundle\Asset\Installer\OutputAwareAssetsInstaller" decorates="sylius.theme.asset.assets_installer" decoration-priority="256">
            <argument type="service" id="sylius.theme.asset.assets_installer.output_aware.inner" />
        </service>

        <service id="sylius.theme.asset.path_resolver" class="Sylius\Bundle\ThemeBundle\Asset\PathResolver" public="false" />

        <!-- Overridden services -->
        <service id="assets.path_package" class="Sylius\Bundle\ThemeBundle\Asset\Package\PathPackage" abstract="true">
            <argument type="service" id="sylius.context.theme" />
            <argument type="service" id="sylius.theme.asset.path_resolver" />
            <argument type="service" id="assets.context" />
        </service>
    </services>
</container>

```

**Comment by venyii** [comment]
This error was already fixed in #8031 (current master)

Duplicates:
https://github.com/Sylius/Sylius/issues/8149
https://github.com/Sylius/Sylius/issues/8174
https://github.com/Sylius/Sylius/issues/8164

**Comment by Mathieudewet** [comment]
Ok thanks
